src/yeahml/build/components/configure.py

```diff
@@ -1,19 +0,0 @@
-import functools
-import types
-
-
-def copy_func(f):
```

Comment:
I think we need to keep this one or `from yeahml.build.components.util import copy_func` -- not sure why I kept two (of the same thing....) but we have to choose one (probably util for now), then correctly link the import in `build/components/activation` and `build/components/loss`

to either

`from yeahml.build.components.util import copy_func`
or
`from yeahml.build.components.configure import copy_func`